demo/util/fps-display.mjs

```diff
@@ -0,0 +1,47 @@
+import * as Util from './util.mjs';
+
+const TEMPLATE = `
+<style>
+  .fps { width: 23%; float: left; text-align: right; padding-right: 2% }
+</style>
+
+  <div id=fps>
+    <div class=fps><span class=place>0</span> FPS</div>
+    <div class=fps><span class=place>0</span> FPS</div>
+    <div class=fps><span class=place>0</span> FPS</div>
+    <div class=fps><span class=place>0</span> FPS</div>
+  </div>
+`;
+
+const MS_PER_SECOND = 1000;
+
+/**
+ * Displays frames-per-second performance of the page for recent frames.
+ */
+export class FpsDisplay extends HTMLElement {
+  #lastTime = null;
+  #fps
```

Comment:
Missing semicolon

Reply:
Fixed. Odd that eslint doesn't find this. I think it's similar to the other rule I fixed, which has yet to be pulled into head, so I guess I'll leave this.